study.md

```diff
@@ -5,14 +5,44 @@ Briefly describe the similarities and differences between `push`, `pop`,
 
 Include differences in return value and chain-ablilty
 
-| method                  | Differences | Similarities |
-|-------------------------|-------------|--------------|
-| `push`                  |             |              |
-| `<<` (compare to`push`) |             |              |
-| `pop`                   |             |              |
-| `unshift`               |             |              |
-| `shift`                 |             |              |
+push (ruby):
+a = [6, 7, 8]
+a.append(9)
```

Comment:
`.append` does not exist in Ruby, it only exists in jQuery: http://api.jquery.com/append/